client/my-sites/upgrades/domain-management/email/add-google-apps-card.jsx

```diff
@@ -29,14 +32,13 @@ const AddGoogleAppsCard = React.createClass( {
 	mixins: [ analyticsMixin( 'domainManagement', 'email' ) ],
 
 	render() {
-		const gapps = this.props.products.gapps,
+		const prices = get( this.props, 'products.gapps.prices', [] ),
```

Comment:
@aidvu I've done one better and did it all in one step thanks to the alternative syntax for [lodash `get`](https://lodash.com/docs#get) (using an array, instead of string). I've used 0 as the default - technically, if there's no price for that currency, I could default to the USD price, but that seems very unlikely, especially with @delputnam's patch.